docs/changelog.md

```diff
@@ -2,6 +2,28 @@
 title: Changelog
 ---
 
+### 0.15.25
+* [FEATURE] Since value set in expectation kwargs is list of strings, do not emit expect_column_values_to_be_in_set for datetime valued columns (#6046)
+* [FEATURE] add failed expectations list to slack message (#5812) (thanks @itaise)
+* [FEATURE] Enable only ExactNumericRangeEstimator and QuantilesNumericRangeEstimator in "datetime_columns_rule" of OnboardingDataAssistant (#6063)
+* [BUGFIX] numpy typing behind `if TYPE_CHECKING` (#6076)
```

Comment:
This bug was never part of a release. Do we still want to include this in the change log?

Reply:
I tagged the latest commit on `develop` so I believe it should stay!